AGPs/AGP-LD2.md

```diff
@@ -0,0 +1,24 @@
+---
+AGP: LD2
+Title: Allow Finance proposals that could otherwise fit in Nest or Flock
+Author: lkngtn (@lkngtn)
+Status: Draft
+Track: Association
+Created: 2019-10-02
+---
+
+# AGP-LD2: Allow Finance proposals that could otherwise fit in Nest or Flock
+
+## Description of desired Association policy change
+
+During ANV-3 AGP-72 was rejected by the Association with the following reasoning:
+
+> Historically, the Aragon Association board has been rejecting Finance Track AGPs that could fit in the Nest or Flock grant programs instead. The Nest and Flock programs are the only two ways the Aragon Association currently manages grants programs, and to fund something outside of them, when they could fit there, would create a bad precedent. Should there be a need for a different funding mechanism for specific projects, an AGP should be created to set the guidelines, budget and governance for such a program.
+
+Source: https://forum.aragon.org/t/agp-reviews-for-anv-3-by-the-aragon-associations-board/1117
+
+This proposal is to change the Association Policy on rejecting these types of proposals on this basis. It would still allow the Association to use its discretion on approving or rejecting proposals, but proposals should not be rejected on the basis that they would otherwise fit into a different program that is outside of the AGP process.
+
+## Motivation for changing this Association policy
+
+Both Flock and Nest are useful programs, and it may be reasonable to divert Finance proposals that would be a good fit to those programs away from the AGP process. However, doing significantly reduces the flexibility and power of Finance Track AGPs. This proposal gives ANT holders an opportunity to weigh in on this policy change, rejecting this proposal would indicate that ANT holders would like the AA to continue to reject proposals if they would otherwise fit into Flock, Nest or a similar program, and approving this proposal would require the AA the to change this policy of rejecting on the basis that a proposal could potentially go through Flock, Nest or a similar program.
```

Comment:
```suggestion
Both Flock and Nest are useful programs, and it may be reasonable to divert Finance proposals that would be a good fit to those programs away from the AGP process. However, doing significantly reduces the flexibility and power of Finance Track AGPs. This proposal gives ANT holders an opportunity to weigh in on this policy change, rejecting this proposal would indicate that ANT holders would like the AA to continue to reject proposals if they would otherwise fit into Flock, Nest or a similar program, and approving this proposal would require the AA the to change this policy of rejecting on the basis that a proposal could potentially go through Flock, Nest or a similar program.


## License
Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
```